utils/soldier_builder.py

```
@@ -0,0 +1,16 @@
+from models.soldier import Soldier
+from tests import last_names, soldier
+
+
+class SoldierBuilder:
+
+    @staticmethod
+    def build_soldier(soldier):
```

Ruff (0.12.2)

Redefinition of unused `soldier` from line 2

(F811)